core/templates/dev/head/directives.js

```diff
@@ -162,3 +162,29 @@ oppia.directive('mobileFriendlyTooltip', ['$timeout', function($timeout) {
     }
   };
 }]);
+
+oppia.directive('scrollOnClick', function() {
```

Comment:
Just clarified with @BenHenning -- we should not be adding new directives to directives.js. Instead, you should put it into a new file in core/templates/dev/head/domain/dashboard/, for example core/templates/dev/head/domain/dashboard/ScrollOnClickDirective.js

This is because this scroll directive is very specific to the dashboard dropdown.

Sorry for not picking up on this earlier!